codegen/tests/simplegrpc/simplegrpc_test.go

```diff
@@ -5,8 +5,8 @@ import (
 	"testing"
 	"time"
 
-	pb "github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/grpc"
 	"github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/simple"
+	pb "github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/simplepb"
```

Comment:
This commit should have been merged with the previous one, if this renaming of simplegrpc -> simplepb is really needed.

Reply:
Will be doing a merge commit here :)